modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -2339,23 +2340,51 @@ public void testUpdateCustomObjectDefinition() throws Exception {
 				true
 			).build());
 
-		objectDefinition =
-			_objectDefinitionLocalService.updateCustomObjectDefinition(
-				null, objectDefinition.getObjectDefinitionId(), 0,
-				objectField.getObjectFieldId(), 0,
-				objectField.getObjectFieldId(), false,
-				objectDefinition.isActive(), objectDefinition.getClassName(),
-				true, false, true, false, false, false,
-				LocalizedMapUtil.getLocalizedMap("Able"), "Able", null, null,
-				false, LocalizedMapUtil.getLocalizedMap("Ables"),
-				objectDefinition.getScope(), objectDefinition.getStatus());
+		Locale defaultLocale = LocaleUtil.getDefault();
```

Comment:
@pedro-oliveira446 in many places in the portal this variable is defined as `defaultLocale` or `locale`, but I believe we should define it as `locale` because there is no name conflict.